### ISSUE TITLE
add occurrenceID to the Identification History extension

### ISSUE THREAD

**Issue by sformel-usgs** [gbif/rs.gbif.org#147]
I reached out to @timrobertson100 and he asked me to pull this issue.

Like #136 , it would be useful to add `occurrenceID` to the '[Identification History](https://rs.gbif.org/extension/identification_history_2024-02-19.xml)` extension.  This would be especially useful for the eDNA community, who are concerned about the ability to track taxonomic updates as reference databases improve. Likewise, it would give GBIF a structure to store taxonomic updates, if GBIF decides to reinterpret DNA sequences as a service to the community.

Tagging  @tobiasgf, @ksilnoaa, @saarasuominen, @eboulanger, @MathewBiddle for awareness.

**Comment by dagendresen** [comment]
For eDNA identification history, would not adding [materialEntityID](http://rs.tdwg.org/dwc/terms/materialEntityID) make even more sense?

**Comment by timrobertson100** [comment]
> For eDNA identification history, would not adding [materialEntityID](http://rs.tdwg.org/dwc/terms/materialEntityID) make even more sense?

I can understand this, but while there is no core entity for Material and GBIF/OBIS only index Occurrence records I don't think it would be a practical approach. This is an example of something that the new data model will properly fix; i.e. distinguishing between events of organisms occurring (Occurrences), how they were detected/observed, the evidence (e.g. material / media etc) to support the claim and the relationships between entities. Knowing that is emerging and will use different schemas anyway, my feeling is occurrenceID is a pragmatic approach for now within the DwC-A structure.

**Comment by sformel-usgs** [comment]
@dagendresen I agree with Tim that this is a pragmatic solution for testing publishing models of data types like eDNA now, while we work toward the new data model. I look forward to being able to use `materialEntityID` as part of the model as well.
